ui/pages/confirmations/components/confirm/header/header-info.tsx

```diff
@@ -66,16 +68,30 @@ const HeaderInfo = ({
 
   const { balance: balanceToUse } = useBalance(fromAddress);
 
+  const isSignature = isSignatureTransactionType(currentConfirmation);
+
+  const eventProps = isSignature
+    ? {
+        location: MetaMetricsEventLocation.SignatureConfirmation,
+        signature_type: (currentConfirmation as SignatureRequestType)?.msgParams
+          ?.signatureMethod,
+      }
+    : {
+        location: MetaMetricsEventLocation.Transaction,
+        transaction_type: currentConfirmation?.type,
+      };
+
   function trackAccountModalOpened() {
-    trackEvent({
-      category: MetaMetricsEventCategory.Transactions,
+    const event = {
+      category: MetaMetricsEventCategory.InpageProvider,
```

Comment:
what is the reason for updating the category here? It seems we should revert this since we are not calling the event from an in-page provider